test/src/test/java/org/corfudb/runtime/concurrent/CorfuQueueTxTest.java

```diff
@@ -60,7 +60,7 @@ public void TXBegin(TransactionType type) {
         }
     }
 
-    protected final int numIterations = PARAMETERS.NUM_ITERATIONS_MODERATE;
+    protected final int numIterations = 100;
```

Comment:
just out of curiosity, does the test not run fast for 1000 or did you lower it for debugging?

Reply:
The test fidelity doesn't increase from 100 to 1000, that's why I changed it. CI is slow and this test is one of the tests that have been failing intermittently for a long time.